docker/playwright.yml

```diff
@@ -0,0 +1,33 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Playwright base service
+# ----------------------------------------------------------------------------------------------------------------------
+
+services:
+    artemis-playwright:
+        image: mcr.microsoft.com/playwright:v1.40.0-jammy
+        pull_policy: if_not_present
+        environment:
+            baseUrl: "https://artemis-nginx"
+            adminUsername: "${bamboo_artemis_admin_username}"
+            adminPassword: "${bamboo_artemis_admin_password}"
+            username: "${bamboo_playwright_username_template}"
+            password: "${bamboo_playwright_password_template}"
+            allowGroupCustomization: "true"
+            studentGroupName: "artemis-e2etest-students"
+            tutorGroupName: "artemis-e2etest-tutors"
+            editorGroupName: "artemis-e2etest-editors"
+            instructorGroupName: "artemis-e2etest-instructors"
+            createUsers: "${bamboo_playwright_create_users}"
```

Comment:
Is this needed like this? Typically env vars are in all CAPS and word are separated by underscores. So instead of `baseUrl` you'd typically use `BASE_URL`.
As an example heres the Google Style Guide for shell stuff about it: [Link](https://google.github.io/styleguide/shellguide.html#constants-and-environment-variable-names)